data/campaigns/Heir_To_The_Throne/scenarios/17_Scepter_of_Fire.cfg

```diff
@@ -88,6 +90,9 @@
                     [/gold_carryover]
 
                     [note]
+                        [show_if]
+                            {VARIABLE_CONDITIONAL moved_too_close boolean_equals false}
+                        [/show_if]
```

Comment:
OK. That's sort of the process I'm hoping for, anyway.

It's all just having my head on straight when rebasing and pushing. Pencil and paper are your friend!